global_helpers/panther_greynoise_helpers.py

```diff
@@ -0,0 +1,155 @@
+import ast
+import datetime
+
+from dateutil import parser
+from panther_base_helpers import PantherUnexpectedAlert, deep_get
+
+
+class GreyNoiseBasic:
+    def __init__(self, event):
+        self.noise = deep_get(event, "p_enrichment", "greynoise_noise_basic")
+        if self.noise is None:
+            self.advanced_enabled = deep_get(event, "p_enrichment", "greynoise_noise_advanced")
+            if self.advanced_enabled is not None:
+                raise PantherUnexpectedAlert(
```

Comment:
WDYT about saving all exceptions as vars and referencing relevant var when exception needed?

Reply:
See above, I think I'm gonna write brief Exception class exception for this